test/junit/scala/collection/immutable/ListMapTest.scala

```diff
@@ -0,0 +1,16 @@
+package scala.collection.immutable
```

Comment:
Most, if not all, JUnit tests lack a copyright header. I already wrote several tests for this repository before and the copyright never was a requirement... Are you sure it's mandatory?